docs/data/sdks/ios-swift/migration.md

```diff
@@ -242,175 +184,115 @@ The APIs for setting user properties are the same, except for the removal of `in
+-   "membership": "paid",
+-   "payment": "bank",
+- ])
++ let identify = Identify()
++ identify
++   .set(property: "membership", value: "paid")
++   .set(property: "payment", value: "bank")
++ amplitude.identify(identify: identify)
+```
 
 #### `identify()`
 
 You can now make an identify call on `amplitude` without calling `instance()`.
 
-=== "Amplitude-iOS"
-
-    ```swift
-    let identify = AMPIdentify()
-    identify.set("membership", value: "paid")
-    Amplitude.instance().identify(identify)
-    ```
-
-=== "Amplitude-Swift"
-
-    ```swift
-    let identify = Identify()
-    identify.set(property: "membership", value: "paid")
-    amplitude.identify(identify: identify)
-    ```
+```diff
+- let identify = AMPIdentify()
+- identify.set("membership", value: "paid")
+- Amplitude.instance().identify(identify)
++ let identify = Identify()
++ identify.set(property: "membership", value: "paid")
++ amplitude.identify(identify: identify)
+```
 
 ### Set group properties
 
 ### `groupIdentify()`
 
 You can now make an identify call on `amplitude` without calling `instance()`.
 
-=== "Amplitude-iOS"
-
-    ```swift
-    let identify = AMPIdentify()
-    identify.set("membership", value: "paid")
-    Amplitude.instance().groupIdentify(
-      withGroupType: "TEST-GROUP-TYPE", 
-      groupName: "TEST-GROUP-NAME", 
-      groupIdentify: identify
-    )
-    ```
-
-=== "Amplitude-Swift"
-
-    ```swift
-    let identify = Identify()
-    identify.set(property: "membership", value: "paid")
-    amplitude.groupIdentify(
-      groupType: "TEST-GROUP-TYPE", 
-      groupName: "TEST-GROUP-NAME", 
-      identify: identify
-    )
-    ```
+```diff
+- let identify = AMPIdentify()
+- identify.set("membership", value: "paid")
+- Amplitude.instance().groupIdentify(
+-   withGroupType: "TEST-GROUP-TYPE", 
+-   groupName: "TEST-GROUP-NAME", 
+-   groupIdentify: identify
+- )
+
++ let identify = Identify()
++ identify.set(property: "membership", value: "paid")
++ amplitude.groupIdentify(
++   groupType: "TEST-GROUP-TYPE", 
++   groupName: "TEST-GROUP-NAME", 
++   identify: identify
++ )
+```
 
 ### Tracking revenue
 
 #### `logRevenueV2()`
 
 Track revenue using `revenue()` API on `amplitude` without calling `instance()`.
 
-=== "Amplitude-iOS"
-
-    ```swift
-      let revenue = AMPRevenue()
-      revenue.setProductIdentifier("productIdentifier")
-      revenue.setQuantity(3)
-      revenue.setPrice(NSNumber(value: 3.99))
-
-      Amplitude.instance().logRevenueV2(revenue)
-    ```
-
-=== "Amplitude-Swift"
+```diff
+- let revenue = AMPRevenue()
+- revenue.setProductIdentifier("productIdentifier")
+- revenue.setQuantity(3)
+- revenue.setPrice(NSNumber(value: 3.99))
+- Amplitude.instance().logRevenueV2(revenue)
 
-    ```swift
-    let revenue = Revenue()
-    revenue.productId = "123"
-    revenue1.price = 12
-            
-    amplitude.revenue(revenue: revenue)
-    ```
++ let revenue = Revenue()
++ revenue.productId = "123"
```

Comment:
Maybe the snippet should use the same properties/values as in Amplitude-iOS snippet above.

Reply:
Good call. Fixed.